src/stores/webSocket.js

```diff
@@ -6,13 +6,25 @@ import { useTransactionDetailsStore } from '@/stores/transactionDetails'
 
 export const useWebSocket = defineStore('webSocket', () => {
   const { WEBSOCKET_URL } = useRuntimeConfig().public
-  const { processSocketMessage } = useRecentBlocksStore()
+  const { processSocketMessage, updateBlockHeight } = useRecentBlocksStore()
   const { updateTransactionTypeData } = useTransactionDetailsStore()
 
   const webSocket = shallowRef()
+  const isSubscribedToTransactionDetails = ref(false)
   const isSubscribedToKeyblockDetails = ref(false)
   const subscribedTransactionId = ref(null)
 
+  watch(isSubscribedToTransactionDetails, newValue => {
```

Comment:
Is this logic needed? Can't we just leave the subscription open?
When the webSocket is opened the subscription is already triggered:
https://github.com/aeternity/aescan/pull/569/files#diff-6eaf0a1b5ea40a2f808855772c29c950791df43f5d4e386e7e89d8cb3542589bR66
Could this cause some issues?

Reply:
Yep, you are right. Can be simplified .) Thanks for the suggestion. Fixed

Reply:
Looks good 👏